<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,1">
    <item>
     <widget class="QGraphicsView" name="bigView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>420</width>
           <height>207</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,7">
            <item>
             <widget class="QLabel" name="nodeNumberLabel">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>节点数：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="nodeInput">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string/>
              </property>
              <property name="statusTip">
               <string/>
              </property>
              <property name="accessibleName">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>默认值：2000</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,7">
            <item>
             <widget class="QLabel" name="runTimeLabel">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>运行时间：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="runTimeInput">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="placeholderText">
               <string>默认值: 10000s</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,7">
            <item>
             <widget class="QLabel" name="cycleLabel">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>业务周期：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="cycleInput">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="placeholderText">
               <string>默认值：10000s</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,7">
            <item>
             <widget class="QLabel" name="lengthLabel">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>场景长度：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lengthInput">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="placeholderText">
               <string>默认值：40m</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,7">
            <item>
             <widget class="QLabel" name="broadLabel">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>场景宽度：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="broadInput">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="placeholderText">
               <string>默认值：40m</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="4,7">
            <item>
             <widget class="QLabel" name="distanceLabel">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>发送/监听距离：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="distanceInput">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="placeholderText">
               <string>默认值：20m</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,3,3">
          <item>
           <widget class="QPushButton" name="runButton">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>运行</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="cancleButton_2">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>取消</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGraphicsView" name="smallView"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>nodeInput</tabstop>
  <tabstop>runTimeInput</tabstop>
  <tabstop>cycleInput</tabstop>
  <tabstop>lengthInput</tabstop>
  <tabstop>broadInput</tabstop>
  <tabstop>distanceInput</tabstop>
  <tabstop>runButton</tabstop>
  <tabstop>cancleButton_2</tabstop>
  <tabstop>smallView</tabstop>
  <tabstop>bigView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
